src/aldrich/TrouserPattern.java

```diff
@@ -206,7 +208,7 @@ protected void createBlocks() {
         Block backblock = blocks.get(1);
 
         // Adding Step 24
-        backblock.addKeypoint(new Vector2D((d_BodyRise + 0.5), (-((((b_Hips / 12) + 2.0) + ((b_Hips / 16.0) + 1.0) + ((b_Hips / 16.0) + 1.0)) / 2.0 + 0.8))));
+        backblock.addKeypoint(new Vector2D((d_BodyRise + 0.5), -((b_Hips/12)+2+(b_Hips/16)+1+0.8+(((b_Hips/16)+1)/2))));
```

Comment:
As I mentioned in the meeting, introduce as much whitespace as you like to make the code more readable -- try to follow the same formatting guidelines (coding standards) as you see already. it should be impossible to tell stylistically that the code has been written by more than one person.